_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,435 @@
+		>)
+	</p>
+</html>
+```
+
+#### Passed Example 8
+
+The two HTML `span` element have an [explicit role][] of link, the same [accessible name][], the same [context][programmatically determined link context], and lead to the [same resource][].
+
+```html
+<html lang="en">
+	<p>
+		<span
+			role="link"
+			tabindex="0"
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html'"
+		>
+			My university
+		</span>
+
+		<span
+			role="link"
+			tabindex="0"
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html'"
+		>
+			My university
+		</span>
+	</p>
+</html>
+```
+
+#### Passed Example 9
+
+The two SVG `a` elements have the same [accessible name][], same [context][programmatically determined link context] and link to the [same resource][].
+
+```html
+<html lang="en">
+	<p>
+		<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+			<a href="http://facebook.com" aria-label="Follow us">
+				<circle cx="50" cy="40" r="35" />
+			</a>
+
+			<a href="http://facebook.com">
+				<text x="50" y="90" text-anchor="middle">
+					Follow us
+				</text>
+			</a>
+		</svg>
+	</p>
+</html>
+```
+
+#### Passed Example 10
+
+The elements in the set comprised by one SVG `a` element and one HTML `a` element have the same [accessible name][], same [context][programmatically determined link context] and link to the [same resource][].
+
+```html
+<html lang="en">
+	<p>
+		<a href="http://facebook.com">Follow us</a>
+
+		<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+			<a href="http://facebook.com" aria-label="Follow us">
+				<circle cx="50" cy="40" r="35" />
+			</a>
+		</svg>
+	</p>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context] but go to different resources.
+
+```html
+<html lang="en">
+	<p>
+		We are on social media:
+		<a href="http://facebook.com">Follow us</a>
+		<a href="http://twitter.com">Follow us</a>
+	</p>
+</html>
+```
+
+#### Failed Example 2
+
+The two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context]. They link to web pages that are similar, but have different information in their content.
+
+```html
+<html lang="en">
+	<p>
+		Learn more (<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html"
+			>Contact us</a
+		>) and get in touch (
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html"
+			>Contact us</a
+		>)
+	</p>
+</html>
+```
+
+#### Failed Example 3
+
+The two HTML `span` elements have an [explicit role][] of link, same [accessible name][] and [context][programmatically determined link context], but lead to resources that offer different content.
+
```

Comment:
this example seems to me to be "ambiguous to all users"

Reply:
Agree that each link can be ambiguous to all users. Updated the text to be similar to what is used in some of the other examples.